index.html

```diff
@@ -65,6 +65,82 @@ <h2>Introduction</h2>
+      class="attributes">
+        <dt><dfn><code>playoutDelayHint</code></dfn> of type <span
+          class="idlAttrType">double</span>, nullable</dt>
+        <dd>
+          <p>This attribute allows the application to give a hint to the User
+          Agent about what playout delay is acceptable. The User Agent SHOULD
+          NOT playout audio or video that is received unless this amount of time
+          has passed in seconds, allowing the User Agent to perform more or less
+          buffering than it might otherwise do. This allows to influence the
+          tradeoffs between having a higher delay and the risk that buffers such
+          as the jitter buffer will run out of audio or video frames to play due
+          to network jitter. This API only serves to provide a hint, and the
+          User Agent may ignore it. The hint SHOULD NOT be followed if it
+          significantly impacts audio or video quality (e.g. bad network), or if
+          the value implies allocating larger buffers than the User Agent is
+          willing to provide.</p>
+          <p>On getting, this attribute MUST return the value of the
+          <a>[[\PlayoutDelayHint]]</a> internal slot.</p>
+          <p>On setting, the User Agent MUST run the following steps:</p>
+          <ol>
+            <li>
+              <p>Let <var>receiver</var> be the
+              <code><a>RTCRtpReceiver</a></code> object on which the setter is
+              invoked.</p>
+            </li>
+            <li>
+              <p>Let <var>delay</var> be the argument to the setter.</p>
+            </li>
+            <li>
+              <p>If <var>delay</var> is negative, <a>throw</a> an
+              <code>InvalidAccessError</code> and abort these steps.</p>
+            </li>
+            <li>
+              <p>Set the value of <var>receiver</var>'s
+              <a>[[\PlayoutDelayHint]]</a> internal slot to
+              <var>delay</var>.</p>
+            </li>
+            <li>
+              <p>In parallel, begin executing the following steps:</p>
+              <ol>
+                <li>
+                  <p>Update the underlying system about the new delay hint, or
+                  that there is no hint if <var>delay</var> is
+                  <code>null</code>. If at any point in time this value is
+                  larger than the User Agent is able or willing to provide, the
+                  User Agent SHOULD use the maximum value available to match the
+                  requested delay as close as possible. If at any point in time
+                  this value is smaller than the User Agent determines is needed
+                  in order provide sufficient audio or video quality, the User
+                  Agent SHOULD use the smallest value possible.</p>
+                  <p class="note">
+                    If User Agent ignores the hint at any time, this is not
+                    reflected in the <a>[[\PlayoutDelayHint]]</a> internal slot.
```

Comment:
Is there any means of queryin g the value that the User Agent is in fact using?
("no" is a perfectly reasonable answer)

Reply:
No, direct way. But indirectly if you are differentiate "getStats"'s jitterBufferDelay by jitterBufferEmittedCount, then you can observe how values which User Agent in fact using are changing (or not depending on User Agent preference ~ in reality most likely network). We decided to remove paragraph about these stats because it was putting hard assumptions about underlying implementation.

Reply:
The answer is purposefully "no". If the answer was yes that might add fingerprinting surface and might encourage apps to poll it, that's not a rabbit hole I want to go down. We have getStats(), but the less we say about that here the better I think.